### DISCUSSION TITLE
deepin-gtk-theme问题：火狐默认主题下无标题栏下默认按钮过小

### DISCUSSION THREAD

**Discussion by shenmo7192** [linuxdeepin/developer-center#3255] [Features Request & Ideas | 特性请求 & 头脑风暴]
https://github.com/linuxdeepin/deepin-gtk-theme/pull/18/

我曾经以为这个问题解决了，但是实际上只是火狐新的主题覆盖了

使用系统主题的时候，这三个按钮仍然非常小

我想使用和系统同步的暗色模式的时候，就不得不忍受极小的按钮

**Comment by zccrs** [comment]
这实际上算是bug。建议把PR重新打开 @shenmo7192

**Reply by shenmo7192** [reply]
> 这实际上算是bug。建议把PR重新打开 @shenmo7192

reopend

这个样式似乎也不是很好看，但是总比原版那个超微版本的好
